tracee/tracee.go

```diff
@@ -354,17 +355,24 @@ func (t *Tracee) initBPF(ebpfProgram string) error {
 		return fmt.Errorf("error initializing file_writes perf map: %v", err)
 	}
 
+	// To avoid blocking of processEvents or processFileWrites, buffer error channel
+	t.errorsChannel = make(chan error, 2)
+
 	return nil
 }
 
 // Run starts the trace. it will run until interrupted
 func (t *Tracee) Run() error {
 	sig := make(chan os.Signal, 1)
 	signal.Notify(sig, os.Interrupt)
+	done := make(chan struct{})
+	// Signal pipeline that Tracee exits by closing the done channel
+	defer close(done)
```

Comment:
the rest of the "closing code" is at the end of this loop (right after `<-sig`, better to keep all of this code together

Reply:
Will do